python/lsst/ts/externalscripts/base_make_calibrations.py

```diff
@@ -452,10 +453,19 @@ async def call_pipetask(self, image_type, exposure_ids_dict):
         else:
             raise RuntimeError("Nonvalid instrument name: {self.instrument_name")
 
+        # Use the camera-agnostic yaml file if the camera-specific
+        # file does not exist.
+        cp_pipe_dir = os.environ.get("CP_PIPE_DIR")
+        pipeline_yaml_file = f"{cp_pipe_dir}/pipelines/"+f"{pipeline_instrument}/{pipe_yaml}"
+        file_exists = os.path.exists(pipeline_yaml_file)
+        if file_exists:
+            pipeline_yaml_file = "${CP_PIPE_DIR}/pipelines/"+f"{pipeline_instrument}/{pipe_yaml}"
```

Comment:
If the file exists, you don't need to reset it here, do you?  Also, I think this has retained bash style variables that I don't think will work.